lib/expression.js

```diff
@@ -248,7 +253,11 @@ CronExpression._parseField = function _parseField (field, value, constraints) {
     var atoms = val.split(',');
     if (atoms.length > 1) {
```

Comment:
Actually, I was playing around with this validation and I found that we can use a little bit simpler approach here. Let me know what you're thinking about this.

```suggestion
    if (!atoms.every(function (atom) {
      return atom.length > 0;
    })) {
      throw new Error('Invalid list value format');
    }

    if (atoms.length > 1) {
```

Reply:
LGTM